web/src/views/SubmissionPortal/HarmonizerView.vue

```diff
@@ -339,6 +342,7 @@ export default defineComponent({
         </v-icon>
         Go to previous step
       </v-btn>
+      <div v-html="helpHtml" />
```

Comment:
We could replicate [the HTML building from DataHarmonizer](https://github.com/cidgoh/DataHarmonizer/blob/b7cfae38b97a063bd1baa321314f72d8fbad0781/script/main.js#L555-L564) rather than setting the HTML directly.